person.rb

```diff
@@ -0,0 +1,22 @@
+# Person class
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permision: true)
+    @id = id
```

Comment:
-  @Dachrono, I think since the person's id won't be provided during creation, we can assign random numbers to the id. One way to do it is my using the `rand` method. Hints

`@id = Random.rand(1..1000)`

Reply:
tky for your advise I will keep it in mind.